Source/WebCore/Modules/webxr/WebXROpaqueFramebuffer.cpp

```diff
@@ -100,8 +100,7 @@ void WebXROpaqueFramebuffer::startFrame(const PlatformXR::Device::FrameData::Lay
     auto bufferHeight = static_cast<uint32_t>(size.height());
 
     auto gCGL = static_cast<GraphicsContextGLCocoa*>(m_context.graphicsContextGL());
-    GCGLenum textureTarget = gCGL->drawingBufferTextureTarget();
-    GCGLenum textureTargetBinding = gCGL->drawingBufferTextureTargetQueryForDrawingTarget(textureTarget);
+    auto [textureTarget, textureTargetBinding] = gl.drawingBufferTextureTargetAndBinding();
```

Comment:
"drawing buffer" refers to the WebGL drawing buffer, which by manner of implementation legacy is implemented as GraphicsContextGL default frame buffer.

Here we are not interested in "drawing buffer texture target", as that is not a concept for an arbitrary framebuffer object. Instead, we are interested in question:
"To what texture target I need to bind a texture in order to bind an external image to the texture?"
Thus your function could be called "externalImageTextureBindingPoint()".

Internally to GraphicsContextGLANGLE, drawingBufferTextureTargetAndBinding could call your new function externalImageTextureBindingPoint() for Cocoa.

The implementation modifications are fine.